src/components/posts/PostItem.js

```diff
@@ -124,8 +126,17 @@ const PostItem = ({ post, userId, setPostResponse, setPost, setUser }) => {
             {post.user.profile.firstName} {post.user.profile.lastName}
           </h3>
         </div>
-
+        <div>
+          {post.likes.length > 10 ? 
```

Comment:
Could make a constant variable for this 10. That way it is easier to understand what its there for